### PR TITLE
[Keymap] Adding buhearns keymap

### PR DESCRIPTION
**Pull request by buhearns**
## Description

Adding buhearns personal keymap.

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/buhearns/keymap.c [hunk 1]
```diff
@@ -0,0 +1,195 @@
+#include QMK_KEYBOARD_H
+#include "muse.h"
+
+extern keymap_config_t keymap_config;
+
+#define B_ENT FUNC(0) // Tap for enter, hold for right shift
```
Comment (fauxpark) [inline keyboards/planck/keymaps/buhearns/keymap.c]: You can define this as `MT(MOD_RSFT, KC_ENT)` and remove the `fn_actions` section below (it is deprecated).

Comment (buhearns) [reply keyboards/planck/keymaps/buhearns/keymap.c]: Ah, thank you! That makes sense. I tried it with `LT(MOD_RSFT, KC_ENT)` and it obviously didn't work.

File: keyboards/planck/keymaps/buhearns/keymap.c [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+      }
+      break;
+    case B_X113:
+      if (record->event.pressed) {
+        SEND_STRING("sudo systemctl enable x11vnc.service" SS_TAP(X_ENTER));
+      }
+      break;
+    case B_DEAB:
+      if (record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_RALT) SS_TAP(X_KP_0) SS_TAP(X_KP_1) SS_TAP(X_KP_9) SS_TAP(X_KP_6) SS_UP(X_RALT));
```
Comment (drashna) [inline keyboards/planck/keymaps/buhearns/keymap.c]: If you  enable unicode, you could handle this much easier.

https://docs.qmk.fm/#/feature_unicode

Comment (buhearns) [reply keyboards/planck/keymaps/buhearns/keymap.c]: Thank you! I already tried that but didn't realize i had to switch to the UC_WINC mode (as opposed to the UC_WIN mode) for the characters to work.

Everything working now but the Euro sign (0x20AC) and Bullet (0x2022). Am I missing something?

**Comment by buhearns** [comment]
Thank you! Proposed changes have been implemented.

Everything working now with exception of the Euro sign (0x20AC) and Bullet (0x2022). Am I missing something? They should be within the normal Unicode range.

**Comment by drashna** [comment]
Odd that those unicode are not working.  I'm not sure why. They should (and I have tested on my ergodox EZ). 
€ and •

I can merge this now, or wait, just let me know what you'd like us to do.

**Comment by buhearns** [comment]
Yes, please merge it. I am a bit busy the next few weeks but am planning to troubleshoot it sometime. It could very well be something on my end.

And thank you for taking the time, with the testing and all. Cheers!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**